tests/unit/django_spanner/models.py

```diff
@@ -0,0 +1,64 @@
+"""
+Different models used for testing django-spanner code.
+"""
+import os
+from django.db import models
+import django
+from django.db.models import Transform
+from django.db.models import CharField, TextField
```

Comment:
It seems odd to have these imports when you already have: `from django.db import models`
I would remove these and just use `models.Transform`, `models.CharField`, etc.

Reply:
cleaned it up.